features/enable_fips_cloud.feature

```diff
@@ -389,3 +389,52 @@ Feature: FIPS enablement in cloud based machines
            | bionic  | FIPS Updates | fips-updates |https://esm.ubuntu.com/fips/ubuntu bionic/main |
            | focal   | FIPS         | fips         |https://esm.ubuntu.com/fips/ubuntu focal/main  |
            | focal   | FIPS Updates | fips-updates |https://esm.ubuntu.com/fips/ubuntu focal/main  |
+
+    @series.lts
+    @uses.config.machine_type.any
+    @uses.config.machine_type.aws.generic
```

Comment:
any reason we should run this only on AWS?

Reply:
It is just that this scenario is not cloud related and I just chose one to test it on. But if you prefer, I can test it on all clouds we support

Reply:
I'm completely fine with that, thanks